app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
+        ):
+            file_out.write(file_in.read())
+        os.remove(command[1])
```

Comment:
You're creating a new file, copying the contents of the old file into it, and then deleting the old file. This is essentially what moving a file is, but as I mentioned earlier, you can use `shutil.move()` to do this in one step. This will be more efficient and less error-prone.